src/App.jsx

```diff
@@ -9,7 +9,9 @@ function App() {
   const fetchData = async () => {
     const url = `https://api.airtable.com/v0/${
       import.meta.env.VITE_AIRTABLE_BASE_ID
-    }/${import.meta.env.VITE_TABLE_NAME}`;
+    }/${
+      import.meta.env.VITE_TABLE_NAME
+    }?view=Grid%20view&sort[0][field]=title&sort[0][direction]=asc`; //?sort%5B0%5D%5Bfield%5D=title&sort%5B0%5D%5Bdirection%5D=asc;
```

Comment:
Since the URL now works with the non-encoded bracket characters (`[` and `]`) in place, I would recommend removing the commented part of this line.

Reply:
sure merged with PR https://github.com/umavenki/react-todo/pull/13